prov/efa/src/rxr/rxr_pkt_cmd.h

```diff
@@ -65,6 +65,20 @@ void rxr_pkt_handle_recv_completion(struct rxr_ep *ep,
 
 ssize_t rxr_pkt_wait_handshake(struct rxr_ep *ep, fi_addr_t addr, struct rxr_peer *peer);
 
+ssize_t rxr_pkt_trigger_handshake(struct rxr_ep *ep,
+				  fi_addr_t addr, struct rxr_peer *peer);
+
+static inline void queue_tx_ctrl_pkt(struct rxr_ep *ep,
+				     struct rxr_tx_entry *tx_entry,
+				     int ctrl_type)
```

Comment:
On second thought, this function need to be in rxr.h near ```rxr_ep_tx_entry_init```.

Reply:
I did not found `rxr_ep_tx_entry_init ` in `rxr.h`. Are you referring to `rxr_ep_peer_init_tx`

Reply:
I meant ```rxr_tx_entry_init```

Reply:
queueing logic no longer used, resolving.